Library/Formula/libharu.rb

```diff
@@ -13,22 +12,40 @@ class Libharu < Formula
     sha1 "0a20c683544726d870f53c005f79a730b1effffe" => :mountain_lion
   end
 
-  depends_on 'libpng'
-
-  # Fixes compilation against LibPNG 1.5. Can be removed on next release.
-  # Based on a commit in the LibHaru repository which does not apply cleanly
-  # due to a missing CHANGES file:
-  # https://github.com/tony2001/libharu/commit/e5bf8b0.patch
-  patch do
-    url "http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/media-libs/libharu/files/libharu-2.2.1-libpng-1.5.patch?revision=1.1"
-    sha1 "463641d4570d59d632c1878597253db962129599"
-  end
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libpng"
 
   def install
+    system "./buildconf.sh", "--force"
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--with-png=#{Formula["libpng"].opt_prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include "hpdf.h"
+
+      int main(void)
+      {
+        int result = 1;
+        HPDF_Doc pdf = HPDF_New(NULL, NULL);
+
+        if (pdf) {
+          HPDF_AddPage(pdf);
+
+          if (HPDF_SaveToFile(pdf, "test.pdf") == HPDF_OK)
+            result = 0;
+
+          HPDF_Free(pdf);
+        }
+
+        return result;
+      }
+    EOS
+    system ENV.cc, "test.c", "-lhpdf", "-lz", "-o", "test"
```

Comment:
The goal is check for zero result code, output is not relevant. There may be a cleaner way to express that.